VMS/Shared/LoginDisplay.razor

```diff
@@ -1,19 +1,34 @@
 ﻿<AuthorizeView>
     <Authorized>
-        <li class="nav-item">
-            <a href="" class="text-decoration-none">
-                <span class="material-icons my-notifications" data-icon="notifications_none"></span>
-            </a>
-        </li>
-        <li class="nav-item text-decoration-none">
-            <a href="">
-                <span class="material-icons-outlined my-person" data-icon="account_circle"></span>
+        <li class="user-section nav-item dflex flex-column align-items-center position-relative">
+            <a href="" class="d-block text-decoration-none user-section__link">
+                <div class="user-avatar-wrapper position-relative">
+                    <img src="img/hsv.png" alt="user avatar" class="user-avatar">
+                </div>
+                <span class="user-label mt-2 fw-bold">Trang cá nhân</span>
             </a>
+            <span class="material-icons-outlined position-absolute user-section__dropdown-icon">
+                arrow_drop_down
+            </span>
+            <div class="user-section__list-wrapper position-absolute">
+                <ul class="user-section__list list-unstyled bg-white">
+                    <li class="text-center">
+                        <a href="https://shopee.vn/" class="px-3 pt-3 pb-2 text-decoration-none fw-bold user-section__list__profile">Trang cá nhân</a>
```

Comment:
Don't include external link 🤣
Use a temporary internal url instead.

Reply:
Shopee is deleted 🤣 and use @Routes.User instead